app/src/main/java/me/capcom/smsgateway/modules/gateway/GatewayApi.kt

```
@@ -142,7 +142,9 @@ class GatewayApi(
     data class DeviceRegisterRequest(
         val name: String,
         val pushToken: String?,
-    )
+        val simCards: List<SimCard>,
+    ) {
+    }
```

detekt (1.23.8)

[warning] The class or object DeviceRegisterRequest is empty.

(detekt.empty-blocks.EmptyClassBlock)